.config/ansible-lint.yml

```diff
@@ -1,5 +1,9 @@
+---
+profile: production
+strict: true
 skip_list:
   - meta-no-info  # meta/main.yml should contain relevant info.
+  - latest[git]
```

Comment:
Instead of adding a global rule for this, can you add a `noqa latest` on the task that's failing? It looks like it only needs to be added here: https://github.com/redhat-cop/cloud.azure_ops/pull/41/files#diff-a662da4045828f7431b4f1b161d03e1e0cd9936e3e144be10c4aa219350ad1b2L34